system/HTTP/CLIRequest.php

```diff
@@ -172,17 +170,17 @@ protected function parseCommand()
                 if ($optionValue) {
                     $optionValue = false;
                 } else {
-                    $this->segments[] = filter_var($arg, FILTER_SANITIZE_STRING);
+                    $this->segments[] = esc(strip_tags($arg));
```

Comment:
This has nothing to do with 8.1 support.
Isn't the existing implementation that does this kind of conversion a bug?

> This is a very dubious filter that has almost no use. It removes NUL bytes, encodes single and double quotes in HTML, and removes anything between < and optional >. It does not function the same as strip_tags() as pointed out in this comment https://www.php.net/manual/en/filter.filters.sanitize.php#118186
https://wiki.php.net/rfc/deprecations_php_8_1

Reply:
The point of this change is to move away from the use of FILTER_SANITIZE_STRING, which use will trigger a deprecation in 8.1, while still retaining the behavior this flag is supposed to bring.

Reply:
Sorry, my English was bad.

I wanted to say, the behavior this flag is supposed to bring is a bug?
`$arg` is a param from CLI, isn't it? It is not HTML.

Reply:
That was my confusion too when I stumbled here.  Maybe it was assumed you can pass HTML tags in the command line. 🧐 Well I just kinda went along with it.

Reply:
Okay, it is another issue. I will think it again later. 

This PR is to support 8.1. This change preserves the existing (buggy) behavior as much as possible.